cmake/find/amqpcpp.cmake

```diff
@@ -9,6 +9,14 @@ if (NOT ENABLE_AMQPCPP)
     return()
 endif()
 
+
+if (NOT EXISTS "${ClickHouse_SOURCE_DIR}/contrib/libuv")
+    message (ERROR "submodule contrib/libuv is missing. to fix try run: \n git submodule update --init --recursive")
+    message (${RECONFIGURE_MESSAGE_LEVEL} "Can't find internal libuv needed for AMQP-CPP library")
+    set (USE_AMQPCPP 0)
+    return()
+endif()
```

Comment:
Do you consider extracting this code into `cmake/find/libuv.cmake` and running `libuv.cmake` before `amqpcpp.cmake` and `cassandra.cmake`?

Reply:
Add a libuv.cmake and then include libuv.cmake in  amqpcpp.cmake and cassandra.cmake。

@tavplubix